docs/processors/referencing-fields.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Referencing record fields'
+sidebar_position: 1
+---
+
+Very often, processors can be configured to work on a specific field in a
```

Comment:
I think we should also mention that these references are how the fields would be addressed in a Go template, and maybe add a link for go templates format